requirements/base.txt

```diff
@@ -319,7 +322,7 @@ rsa==4.9
     # via google-auth
 selenium==3.141.0
     # via apache-superset
-shillelagh[gsheetsapi]==1.2.10
+shillelagh[gsheetsapi]==1.2.18
```

Comment:
Happy to! But this is a feature that long existed (since I created the SIP), but there was a regression in the API — apparently Google stopped returning a status code of 200 when authorization was needed, and started returning a 403, which ironically broke the flow.